src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,18 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: complete
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
## `foundationReference`
- [ ] @matentzn or @twhetzel  advise
<details><summary>Details</summary>
<p>

In the [property discussion thread](https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1483728680), I wrote:

> include but also put inside `rdfs:comment`?

**What to do?**
What should I do in this case? Should I just ignore it? Should I integrate it into Mondo somehow, e.g. by including in `rdfs:comment`?

**Analysis?**
Perhaps I need to show some cases of how this is used / what it looks like? Is this plan correct?:
- [ ] Show an example @joeflack4 
- [ ] Then, Nico/Trish will advise

</p>
</details>

Reply:
Examples?

Reply:
@matentzn @twhetzel Alright, so it looks like `foundationReference` works something like this. It exists only on annotation axioms for `annotatedProperty` `icd11.foundation:inclusion` and `icd11.foundation:exclusion`.

<details><summary>How I determined it was annotating only these 2</summary>
<p>

1. `cat tmp/component-download-icd11foundation.owl.owl | grep foundationReference -B 2 > ~/Desktop/foundationRef.txt`
2. `cat ~/Desktop/foundationRef.txt| grep annotatedProperty > ~/Desktop/foundationrefprops.txt`
3. Looked at the unique values.

</p>
</details> 

As I wrote in #462:
> I'm not 100% sure, but at face these appear to me to be a way, to add additional modifications beyond a normal subclass relationship. You mark something a subclass of another, but then it might inherit some other relationships which then you can mark with `exclusion`. And then perhaps you can mark `inclusion` for some other classes that do not have a subclass relationship.

<details><summary>Example usage in OWL</summary>
<p>

For the first class, I left all properties just for reference. It actually had 2 axioms annotating its exclusions. I left one out. The axiom that I left in actually refers to the 2nd class I've shown an example for. It also has 1 exclusion and an axiom annotation on it, referenced by the 3rd class I've put as an example, which itself has 4 exclusions, each with an annotation axiom.

```owl
    <owl:Class rdf:about="http://id.who.int/icd/entity/1000337196">
        <rdfs:subClassOf rdf:resource="http://id.who.int/icd/entity/1632259883"/>
        <schema:browserUrl rdf:resource="https://icd.who.int/dev11/f/en#/http%3A%2F%2Fid.who.int%2Ficd%2Fentity%2F1000337196"/>
        <schema:exclusion xml:lang="en">Catatonia</schema:exclusion>
        <schema:exclusion>Delirium</schema:exclusion>
        <schema:inclusion xml:lang="en">Semicoma</schema:inclusion>
        <skos:altLabel xml:lang="en">Semicoma</skos:altLabel>
        <skos:altLabel xml:lang="en">stuporous</skos:altLabel>
        <skos:definition xml:lang="en">Total or nearly total lack of spontaneous movement and marked decrease in reactivity to environment.</skos:definition>
        <skos:prefLabel xml:lang="en">Stupor</skos:prefLabel>
    </owl:Class>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://id.who.int/icd/entity/1000337196"/>
        <owl:annotatedProperty rdf:resource="http://id.who.int/icd/schema/exclusion"/>
        <owl:annotatedTarget xml:lang="en">Catatonia</owl:annotatedTarget>
        <schema:foundationReference rdf:resource="http://id.who.int/icd/entity/486722075"/>
    </owl:Axiom>
    ...



    <owl:Class rdf:about="http://id.who.int/icd/entity/486722075">
        ...
        <schema:exclusion>Harmful effects of drugs, medicaments or biological substances, not elsewhere classified</schema:exclusion>
        <skos:prefLabel xml:lang="en">Catatonia</skos:prefLabel>
    </owl:Class>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://id.who.int/icd/entity/486722075"/>
        <owl:annotatedProperty rdf:resource="http://id.who.int/icd/schema/exclusion"/>
        <owl:annotatedTarget>Harmful effects of drugs, medicaments or biological substances, not elsewhere classified</owl:annotatedTarget>
        <schema:foundationReference rdf:resource="http://id.who.int/icd/entity/1868408442"/>
    </owl:Axiom>



    <owl:Class rdf:about="http://id.who.int/icd/entity/1868408442">
        ...
        <schema:exclusion xml:lang="en">Alcohol intoxication</schema:exclusion>
        <schema:exclusion>Allergic or hypersensitivity conditions</schema:exclusion>
        <schema:exclusion>Disorders due to substance use or addictive behaviours</schema:exclusion>
        <schema:exclusion>Reactions or intoxications due to drugs administered to fetus or newborn</schema:exclusion>
        <schema:inclusion xml:lang="en">overdose of these substances</schema:inclusion>
        <schema:inclusion xml:lang="en">wrong substance given or taken in error</schema:inclusion>
        ...
        <skos:prefLabel xml:lang="en">Harmful effects of drugs, medicaments or biological substances, not elsewhere classified</skos:prefLabel>
    </owl:Class>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://id.who.int/icd/entity/1868408442"/>
        <owl:annotatedProperty rdf:resource="http://id.who.int/icd/schema/exclusion"/>
        <owl:annotatedTarget xml:lang="en">Alcohol intoxication</owl:annotatedTarget>
        <schema:foundationReference rdf:resource="http://id.who.int/icd/entity/1339202943"/>
    </owl:Axiom>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://id.who.int/icd/entity/1868408442"/>
        <owl:annotatedProperty rdf:resource="http://id.who.int/icd/schema/exclusion"/>
        <owl:annotatedTarget>Allergic or hypersensitivity conditions</owl:annotatedTarget>
        <schema:foundationReference rdf:resource="http://id.who.int/icd/entity/642618805"/>
    </owl:Axiom>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://id.who.int/icd/entity/1868408442"/>
        <owl:annotatedProperty rdf:resource="http://id.who.int/icd/schema/exclusion"/>
        <owl:annotatedTarget>Disorders due to substance use or addictive behaviours</owl:annotatedTarget>
        <schema:foundationReference rdf:resource="http://id.who.int/icd/entity/1602669465"/>
    </owl:Axiom>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://id.who.int/icd/entity/1868408442"/>
        <owl:annotatedProperty rdf:resource="http://id.who.int/icd/schema/exclusion"/>
        <owl:annotatedTarget>Reactions or intoxications due to drugs administered to fetus or newborn</owl:annotatedTarget>
        <schema:foundationReference rdf:resource="http://id.who.int/icd/entity/142627676"/>
    </owl:Axiom>
```

</p>
</details> 

So, `inclusion` and `exclusion` hold labels for classes. The URI to the foundation class (`foundationReference`) is then annotated on those properties.

Why not just make `inclusion` and `exclusion` point to URIs for the classes instead of their labels? I suppose it probably has something to do with linearization. I assume new class IDs get generated at linearization-time, but the labels remain static.

Reply:
Please make new issue and raise at meeting. Then resolve comment here. In issue, add some concrete examples from the ontology we can discuss!